src/rudderstack.ts

```diff
@@ -2,96 +2,105 @@ import * as RudderAnalytics from 'rudder-sdk-js'
-   * @param payload Additional information passed to identify the user
-   */
-  identifyEvent = (user_id: string, payload: { language: string }) => {
-    RudderAnalytics.identify(user_id, payload)
-    this.has_identified = true
-  }
+    /**
+     * Pushes page view event to Rudderstack
+     *
+     * @param curret_page The name or URL of the current page to track the page view event
+     */
+    pageView(current_page: string, platform = 'Deriv App') {
+        if (this.has_initialized && this.has_identified && current_page !== this.current_page) {
+            RudderAnalytics.page(platform, current_page)
+            this.current_page = current_page
+        }
+    }
 
-  /**
-   * Pushes page view event to Rudderstack
-   *
-   * @param curret_page The name or URL of the current page to track the page view event
-   */
-  pageView(current_page: string, platform = 'Deriv App') {
-    if (this.has_initialized && this.has_identified && current_page !== this.current_page) {
-      RudderAnalytics.page(platform, current_page)
-      this.current_page = current_page
+    setAccountType(account_type: string) {
+        this.account_type = account_type
     }
-  }
 
-  /**
-   * Pushes reset event to rudderstack
-   */
-  reset() {
-    if (this.has_initialized) {
-      RudderAnalytics.reset()
-      this.has_identified = false
+    /**
+     * Pushes reset event to rudderstack
+     */
+    reset() {
+        if (this.has_initialized) {
+            RudderAnalytics.reset()
+            this.has_identified = false
+        }
     }
-  }
 
-  /**
-   * Pushes track events to Rudderstack. When this method is called before `identifyEvent` method is called, the events tracked will be associated with an anonymous ID.
-   * Otherwise, if the events needs to be associated with a user ID, call `identifyEvent` with the user ID passed first before calling this method.
-   *
-   * @param event The event name to track
-   * @param payload Additional information related to the event
-   */
-  track<T extends keyof TEvents>(event: T, payload: TEvents[T]) {
-    if (this.has_initialized && this.has_identified) {
-      try {
-        RudderAnalytics.track(event, payload)
-      } catch (err) {
-        console.error(err)
-      }
+    /**
+     * Pushes track events to Rudderstack. When this method is called before `identifyEvent` method is called, the events tracked will be associated with an anonymous ID.
+     * Otherwise, if the events needs to be associated with a user ID, call `identifyEvent` with the user ID passed first before calling this method.
+     *
+     * @param event The event name to track
+     * @param payload Additional information related to the event
+     */
+    track<T extends keyof TEvents>(event: T, payload: TEvents[T]) {
+        if (this.has_initialized && this.has_identified) {
+            try {
+                RudderAnalytics.track(event, {
+                    account_type: this.account_type,
```

Comment:
this action already done in analytics.ts
` const trackEvent = <T extends keyof TEvents>(event: T, analyticsData: TEvents[T]) => {
        _rudderstack.track(event, { ...coreData, ...analyticsData })
    }`